com.ibm.wala.util/src/main/java/com/ibm/wala/dataflow/graph/DataflowSolver.java

```diff
@@ -138,7 +144,7 @@ private int mapIt(int i, Object[] allVars, Map<Object, V> varMap) {
      * record that variable (n1, in1) is the same as variable (n2,in2), where (x,true) = IN(X) and
      * (x,false) = OUT(X)
      */
-    public void union(Object n1, Object n2) {
+    public void union(@Nullable Object n1, @Nullable Object n2) {
```

Comment:
A bit strange that the tool made these parameters `@Nullable`; I would not have done this since the first thing this function does is assert they are not null.  I think this is a case of the tool propagating `@Nullable` and not seeing any errors, but a human would have done it differently